projects/js-packages/components/components/threats-data-view/styles.module.scss

```diff
@@ -16,6 +17,37 @@
 	.dataviews-view-list .dataviews-views-list__fields {
 		align-items: center;
 	}
+
+	.dataviews-view-table th.dataviews-view-table__actions-column {
+		text-align: left;
+	}
+
+	.dataviews-view-table tr .dataviews-item-actions {
+		gap: 16px;
+
+		.components-button:not( .dataviews-all-actions-button ) {
+			opacity: 1;
+		}
+	  
+		.dataviews-all-actions-button {
+			display: none;
+		}
+	}
+
+	.dataviews-view-table__actions-column .dataviews-item-actions .components-button.is-compact.has-icon {
+		width: fit-content;
+
+		&:focus,
+		&:focus-visible {
+			box-shadow: none;
+		}
```

Comment:
Do we compensate for this with other focus styles?

Reply:
We do not - I wasn't quite sure to what extent we should compensate for all these things directly within the component or whether various parts should be handled in the implementation.

I removed the built in button element pseudo styles here and set the span element and text styles and a hover effect to match the colours in the designs, but there are other pieces in the component where we default to wp-admin style settings. Should we account for all of this here, or should we allow for the default and update everything where it's in use?